modules/rosa-policy-security-regulation-compliance.adoc

```diff
@@ -15,7 +15,14 @@ Red Hat defines and follows a data classification standard to determine the sens
 
 [id="rosa-policy-data-management_{context}"]
 == Data management
-{product-title} (ROSA) uses AWS Key Management Service (KMS) to help securely manage keys for encrypted data. These keys are used for control plane data volumes that are encrypted by default.
+{product-title} (ROSA) uses AWS Key Management Service (KMS) to help securely manage keys for encrypted data. These keys are used for control plane data volumes that are encrypted by default. Persistent volumes (PVs) for customer applications also use AWS KMS for key management.
+
+[IMPORTANT]
+====
+Only persistent volumes (PVs) created from the default storage class are encrypted by default.
+
+PVs created by using any other storage class are only encrypted if the the storage class is configured to be encrypted.
+====
```

Comment:
```suggestion
```

Doesn't apply, since we're not talking about encryption with a _specific_ key. **All** volumes continue to be encrypted, always.